src/renderer/base/RenderSettings.cpp

```diff
@@ -79,15 +80,19 @@ void RenderSettings::MakeAdjustedColorArray() noexcept
 {
     // The color table has 16 colors, but the adjusted color table needs to be 18
     // to include the default background and default foreground colors
-    std::array<COLORREF, 18> colorTableWithDefaults;
+    std::array<COLORREF, 19> colorTableWithDefaults;
     std::copy_n(std::begin(_colorTable), 16, std::begin(colorTableWithDefaults));
     colorTableWithDefaults[AdjustedFgIndex] = GetColorAlias(ColorAlias::DefaultForeground);
     colorTableWithDefaults[AdjustedBgIndex] = GetColorAlias(ColorAlias::DefaultBackground);
 
-    for (auto fgIndex = 0; fgIndex < 18; ++fgIndex)
+    // We need to use TextColor to calculate the bright default fg
+    TextColor defaultFg;
+    colorTableWithDefaults[AdjustedBrightFgIndex] = defaultFg.GetColor(_colorTable, GetColorAliasIndex(ColorAlias::DefaultForeground), true);

```

Comment:
So this should be a `static` function now right?

Reply:
realistically we should just do #11939 instead of solving `GetColor` having to recalculate the bright foreground every time.

Reply:
^ what Dustin said, but we're leaving #11939 as a todo for now